courses/views_test.py

```diff
@@ -486,6 +486,13 @@ def test_201_on_happy_case_with_modules(self):
                 content_type="application/json"
             )
         assert result.status_code == 201, result.content.decode('utf-8')
+        # check that the payload has been sent to edx as json
+        # this means that the requests library will encode it properly
+        # and add the `application/json` content type header
+        call_to_edx = mock_req.post.call_args
+        assert len(call_to_edx) == 2
```

Comment:
The return value from `call_args` is always 2 unless the mock wasn't called yet: https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.call_args